File: C/Arrays/sum.c
// program to add n number of integers

void main() {
    int num, sum=0, i, n;
    printf("How many number do you want to add? : ");
    scanf("%d", &n);
    for (i = 1; i <= n; i++) {
        printf("Enter number %d: ", i);
        scanf("%d", &num);
        sum = sum + num;
    }
    printf("Sum is: %d", sum);
}


//============================================================================
//============================================================================

File: Python/Condition & Looping/sum.py
# program to add n number of integers

sum = 0
totalNumbers = int(input("How many number do you want to add? : "))
for i in range(1, totalNumbers + 1):
    a = int(input(f"Enter number {i}: "))
    sum += a
print("Sum is: ", sum)
